lib/Windowing/InViewList.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { func, node, number } from 'prop-types';
+import { WindowingContext } from './WindowingProvider';
+
+function InViewList({ items, offset, children, itemHeight }) {
+  const [renderIndexes, setRenderIndexes] = useState({});
```

Comment:
`renderIndexes` is the start and end index for what is "in view". E.g. `[0, 10]` will say to render the first 10 items in the overall collection.